bundles/org.openhab.binding.onecta/src/main/resources/OH-INF/i18n/onecta.properties

```diff
@@ -0,0 +1,1234 @@
+thing-type.config.onecta.account.stubdataFile.label = Stub datafile
+thing-type.config.onecta.account.stubdataFile.description = With the stub datafile conecta data input can be simulated. This is just for testing during development.
+
+# channel group types
+
+channel-group-type.onecta.basicGroupGateway.label = Basic
+channel-group-type.onecta.basicGroupGateway.description = BasicGroup related information
+channel-group-type.onecta.basicGroupIndoorUnit.label = Basic
+channel-group-type.onecta.basicGroupIndoorUnit.description = BasicGroup related information
+channel-group-type.onecta.basicGroupWaterTank.label = Basic
+channel-group-type.onecta.basicGroupWaterTank.description = BasicGroup related information
+
+# channel types
+
+channel-type.onecta.gw-daylight-saving-time-enabled.label = Daylight saving time enabled
+channel-type.onecta.gw-daylight-saving-time-enabled.description = Daylight saving time enabled
+channel-type.onecta.gw-is-firmwareUpdateSupported.label = Is firmware update supported
+channel-type.onecta.gw-is-firmwareUpdateSupported.description = Is firmware update supported
+channel-type.onecta.gw-serialnumber.label = Serialnumber
+channel-type.onecta.gw-serialnumber.description = Serialnumber
+
+# channel types
+
+channel-type.onecta.acunit-leavingWaterTemp.label = Leaving Water Temperature
+channel-type.onecta.acunit-leavingWaterTemp.description = The leaving water temperature (readonly)
+channel-type.onecta.acunit-setleavingWaterOffset.label = SetPoint Leaving water offset.
+channel-type.onecta.acunit-setleavingWaterOffset.description = SetPoint Leaving water offset.
+channel-type.onecta.acunit-setleavingWaterTemp.label = SetPoint Leaving water temperature
+channel-type.onecta.acunit-setleavingWaterTemp.description = SetPoint Leaving water temperature
+channel-type.onecta.acunit-targetTemp.label = Target Temperatur
+channel-type.onecta.acunit-targetTemp.description = The set point temperature
+channel-type.onecta.acunit-targetTempMax.label = Target Temperature Max value
+channel-type.onecta.acunit-targetTempMax.description = The target max temperature
+channel-type.onecta.acunit-targetTempMin.label = Target Temperature Min value
+channel-type.onecta.acunit-targetTempMin.description = The target min temperature
+channel-type.onecta.acunit-targetTempStep.label = Target Temperature value step
+channel-type.onecta.acunit-targetTempStep.description = The target temperature value step
+
+# thing types
+
+thing-type.onecta.domesticHotWaterTank.label = Diakin Onecta Domestic HotWater Tank
+thing-type.onecta.domesticHotWaterTank.description = Diakin Onecta Domestic HotWater Tank
+thing-type.onecta.indoorUnit.label = Diakin Onecta Indoor Unit
+thing-type.onecta.indoorUnit.description = Diakin Onecta Indoor Unit
+
+# thing types config
+
+thing-type.config.onecta.domesticHotWaterTank.refreshDelay.label = Refresh delay.
```

Comment:
Can the dot at the end be removed?

Applies to other entries in this file as well.